src/variants/model/VariantsRequest.ts

```diff
@@ -0,0 +1,5 @@
+import VariantsRequestByProductId from './VariantsRequestByProductId';
+
+export default interface VariantsRequest extends VariantsRequestByProductId {
+  skuOrUpc: string;
```

Comment:
Please rename this `inventoryItemValue`. This is what we call it within the API. You will also notice that `inventoryItemKey` is set to `SKU` or `UPC` on a business. This key correlates to this value.